pkg/apis/operator/v1alpha1/tektonconfig_types.go

```diff
@@ -89,6 +89,9 @@ type TektonConfigSpec struct {
 	// Dashboard holds the customizable options for dashboards component
 	// +optional
 	Dashboard Dashboard `json:"dashboard,omitempty"`
+	// DisableRbacClusterWide holds the option to disable/enable creation of RBAC resources
+	// +optional
+	DisableRbacClusterWide bool `json:"disable-rbac-clusterwide,omitempty"`
```

Comment:
how about

```suggestion
	AutoConfigDefaultSA bool `json:"disable-rbac-clusterwide,omitempty"`
```
true/false

cluster wide is known as all other options here are clusterwide. And `RbacClusterwide` sounds a bit developer oriented.

Reply:
if we use `ConfigurePipelineServiceAccount` it indicates only for ServiceAccount but we are disabling other resources also along with `ServiceAccount` 

So how about using this 
`ConfigureRbacResources` :thinking:

Reply:
If we change the field, we might want to change the "json" field as well.

Reply:
Also, we should probably discuss the naming in the [enhancement review](https://github.com/openshift-pipelines/enhancements/pull/3/files)

Reply:
> we should probably discuss the naming in the enhancement review

absolutely. i missed that. 
i shall add a comment on this thread, if it doesn't resolve here, shortly, then we shall take it up there 🧑‍💻

Reply:
> So how about using this
ConfigureRbacResources 🤔

i agree with you, about the `SA` part. 

one comment on `ConfigureRbacResources` is that it could be read as, configuring RBAC by operator for the fucntioning of pipelines, triggers etc (rbac that comes with the components).

`ConfigureDefaultSA` could have been another name, but the `default` sa itself could make this confusing. 😄

Reply:
I think `ConfigureRBACResources` is good enough 👍 
with proper documentation, this name should work fine.

Reply:
> i shall add a comment on this thread, if it doesn't resolve here, shortly, then we shall take it up there technologist

What do you mean ?

Reply:
if we agree on the `ConfigureRBACResources` then let us update that in the enhancement and we shall resolve this. :+1: